btf/btf.go

```diff
@@ -500,6 +500,17 @@ func (s *Spec) TypeByName(name string, typ interface{}) error {
 	return nil
 }
 
+// LoadSplitSpec opens file and calls LoadSplitSpecFromReader on it.
```

Comment:
Nit: the callout to `LoadSplitSpecFromReader` isn't so relevant.

Let's reuse the docstring:

```
// LoadSplitSpec loads split BTF from the given file.
//
// Types from base are used to resolve references in the split BTF.
// The returned Spec only contains types from the split BTF, not from the base.
```

Reply:
Sorry for this.
Thank you for pointing this out, I changed it